disco_aws_automation/disco_aws.py

```diff
@@ -394,6 +394,24 @@ def provision(self, ami, hostclass=None, owner=None, instance_type=None, monitor
 
         elb = self.update_elb(hostclass, update_autoscaling=False, testing=testing)
 
+        tags = {"hostclass": hostclass,
+                "owner": user_data["owner"],
+                "environment": self.environment_name,
+                "chaos": chaos,
+                "is_testing": "1" if testing else "0"}
+
+        if self.vpc.environment_class:
+            tags['environment_class'] = self.vpc.environment_class
```

Comment:
Are sandboxes correctly tagged everywhere? Maybe this should default to `development`.

Reply:
If VPCs are not tagged with `environment_class`, this tag is not added at all for the hostclass. I did not have a real tag value to default to, but I can default it to `development`.